From 1e4dada823ab940b1f733f88755ac52bb1e90594 Mon Sep 17 00:00:00 2001
From: "Marco A. Gutierrez" <marco@openrobotics.org>
Date: Fri, 9 Oct 2020 04:59:39 +0000
Subject: [PATCH 2/2] sourcing foxy not eloquent

---
 debian/rules.em | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/rules.em b/debian/rules.em
index 8dfaa1f..5194d22 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -53,7 +53,7 @@ override_dh_shlibdeps:
 	# In case we're installing to a non-standard location, look for a setup.sh
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
-	if [ -f "/opt/ros/eloquent/setup.sh" ]; then . "/opt/ros/eloquent/setup.sh"; fi && \
+	if [ -f "/opt/ros/foxy/setup.sh" ]; then . "/opt/ros/foxy/setup.sh"; fi && \
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_shlibdeps -l$(CURDIR)/debian/@(Package)/@(InstallationPrefix)/lib/
 
@@ -61,6 +61,6 @@ override_dh_auto_install:
 	# In case we're installing to a non-standard location, look for a setup.sh
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
-	if [ -f "/opt/ros/eloquent/setup.sh" ]; then . "/opt/ros/eloquent/setup.sh"; fi && \
+	if [ -f "/opt/ros/foxy/setup.sh" ]; then . "/opt/ros/foxy/setup.sh"; fi && \
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_install
-- 
2.25.1

